src/Resizer.php

```diff
@@ -165,16 +172,21 @@ protected function createImage(ImageInterface $image, $path)
      * @param ResizeCoordinatesInterface $coordinates
      * @param ResizeOptionsInterface     $options
      *
-     * @return string The realtive target path
+     * @return string The relative target path
      */
     private function createCachePath($path, ResizeCoordinatesInterface $coordinates, ResizeOptionsInterface $options)
     {
         $pathinfo = pathinfo($path);
         $imagineOptions = $options->getImagineOptions();
         ksort($imagineOptions);
 
+        $hashPath = $path;
```

Comment:
If there is no `$baseDir` the path would not be relative, wouldn’t `$relPath` be misleading then?